### DISCUSSION TITLE
How does this project differ from similar popular open source project formbricks?

### DISCUSSION THREAD

**Discussion by Greatz08** [OpnForm/OpnForm#552] [General]
Can you list all unique feature or advantages people will enjoy in this project compared to formbricks. What makes this project truly different from formbricks i wanna know :-))

**Comment by JhumanJ** [comment]
Good question. It seems like both projects were started more or less around the same time, but took slightly different directions. I'm not very familiar with FormBricks but I encourage you to try both and see which tool suits you best!
